problems/2.functions.js

```diff
@@ -20,17 +25,20 @@ function greeting() {
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
-
+function incrementGlobalCount(val) {
+  return global.globalCount() + val
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
global.globalCount += val